TransferList/TransferList.js

```diff
@@ -128,59 +138,60 @@ const TransferListBase = kind({
 	},
 
 	computed: {
-		renderItems: () => ({elements, list, onSelect, selectedItems, ...rest}) => {
-			return elements.map((element, index) => {
-				const selected = -1 !== selectedItems.findIndex((pair) => pair.element === element && pair.list === list);
-
-				const handleClick = useCallback(() => {
-					onSelect(element, index, list);
-				}, [element, index, list, onSelect]); // eslint-disable-line react-hooks/exhaustive-deps
-
-				const handleSpotlightDown = useCallback((ev) => {
-					if (elements.length - 1 !== index) return;
-					ev.preventDefault();
-					ev.stopPropagation();
-				}, [elements, index]); // eslint-disable-line react-hooks/exhaustive-deps
-
-				const handleSpotlightUp = useCallback((ev) => {
-					if (index !== 0) return;
-					ev.preventDefault();
-					ev.stopPropagation();
-				}, [index]);
-
-				return (
-					<CheckboxItem
-						{...rest}
-						draggable
-						className={componentCss.draggableItem}
-						id={`${index}-${list}`}
-						key={index + list}
-						onClick={handleClick}
-						onSpotlightDown={handleSpotlightDown}
-						onSpotlightUp={handleSpotlightUp}
-						selected={selected}
-					>
-						{element}
-					</CheckboxItem>
-				);
-			});
+		renderItem: () => ({elements, list, onSelect, selectedItems, ...rest}) => (data) => {	// eslint-disable-line	enact/display-name
+			const {index, 'data-index': dataIndex} = data;
+			const element = elements[index];
+			const selected = -1 !== selectedItems.findIndex((pair) => pair.element === element && pair.list === list);
+
+			const handleClick = () => {
+				onSelect(element, index, list);
+			};
+
+			const handleSpotlightDown = (ev) => {
+				if (elements.length - 1 !== index) return;
+				ev.preventDefault();
+				ev.stopPropagation();
+			};
+
+			const handleSpotlightUp = (ev) => {
+				if (index !== 0) return;
+				ev.preventDefault();
+				ev.stopPropagation();
+			};
+
+			return (
+				<CheckboxItem
+					{...rest}
+					data-index={dataIndex}
+					draggable
+					className={componentCss.draggableItem}
+					id={`${index}-${list}`}
+					key={index + list}
+					onClick={handleClick}	// eslint-disable-line  react/jsx-no-bind
+					onSpotlightDown={handleSpotlightDown}	// eslint-disable-line  react/jsx-no-bind
+					onSpotlightUp={handleSpotlightUp}	// eslint-disable-line  react/jsx-no-bind
```

Comment:
add `react/jsx-no-bind` to the list on line 1 and remove them from here

Reply:
I would like to keep them like this. The default way to avoid this warning is to wrap the functions in useCallback, but as some of them do change on every rerender there is no point for it here. If we place it as a global rule it will not be enforced where we need it, so it would be better to do it locally.

Reply:
okay